Source/VFECore/Abilities/Ability.cs

```diff
@@ -119,8 +119,12 @@ public virtual bool IsEnabledForPawn(out string reason)
         public float GetAdditionalRadius() =>
             this.def.GetModExtension<AbilityExtension_AdditionalRadius>().GetRadiusFor(this.pawn);
 
-        public virtual float GetPowerForPawn() =>
-            CalculateModifiedStatForPawn(this.def.power, this.def.powerStatFactors, this.def.powerStatOffsets);
+        public virtual float GetPowerForPawn()
+        {
+            float power = CalculateModifiedStatForPawn(this.def.power, this.def.powerStatFactors, this.def.powerStatOffsets);
+            var multiplier = this.def.GetModExtension<AbilityExtension_RandomPowerMultiplier>();
+            return multiplier == null ? power : Rand.Range(power * multiplier.min, power * multiplier.max);
```

Comment:
```suggestion
            return multiplier == null ? power : power * Rand.Range(multiplier.min, multiplier.max);
```
This is more consistent with how vanilla is coded.